src/apps/bridge/sensor_drivers/rbrCodaSensor.cpp

```diff
@@ -193,4 +190,229 @@ RbrCoda_t *createRbrCodaSub(uint64_t node_id, uint32_t rbr_coda_agg_period_ms,
+            printf("Failed to get the length of the node app name\n");
+            break;
+          }
+          char *strbuf = static_cast<char *>(pvPortMalloc(len + 1));
+          configASSERT(strbuf);
+          if (cbor_value_copy_text_string(&node_array_member, strbuf, &len, NULL) !=
+              CborNoError) {
+            break;
+          }
+          strbuf[len] = '\0';
+          printf("RBR Node app name: %s\n", strbuf);
+          vPortFree(strbuf);
+          if (cbor_value_advance(&node_array_member) != CborNoError) {
+            printf("Failed to advance the node array member\n");
+            break;
+          }
+          if (!cbor_value_is_unsigned_integer(&node_array_member)) {
+            printf("expected uint but it is not a string\n");
+            break;
+          }
+          uint64_t gitSha;
+          if (cbor_value_get_uint64(&node_array_member, &gitSha) != CborNoError) {
+            printf("Failed to get the git sha\n");
+            break;
+          }
+          printf("Git SHA: %" PRIx64 "\n", gitSha);
+          if (cbor_value_advance(&node_array_member) != CborNoError) {
+            printf("Failed to advance the node array member\n");
+            break;
+          }
+          if (!cbor_value_is_unsigned_integer(&node_array_member)) {
+            printf("expected uint but it is not a string\n");
+            break;
+          }
+          uint64_t sys_confg_crc;
+          if (cbor_value_get_uint64(&node_array_member, &sys_confg_crc) != CborNoError) {
+            printf("Failed to get the git sha\n");
+            break;
+          }
+          printf("sys_confg_crc: %" PRIx64 "\n", sys_confg_crc);
+          if (cbor_value_advance(&node_array_member) != CborNoError) {
+            printf("Failed to advance the node array member\n");
+            break;
+          }
+          if (!cbor_value_is_map(&node_array_member)) {
+            printf("expected map but it is not a map\n");
+            break;
+          }
+          CborValue sys_cfg_map;
+          if (cbor_value_enter_container(&node_array_member, &sys_cfg_map) != CborNoError) {
+            printf("Failed to enter the sys_cfg_map\n");
+            break;
+          }
+          // Loop through the map to find the rbrCodaType config
+          while (!cbor_value_at_end(&node_array_member)) {
+            if (!cbor_value_is_text_string(&sys_cfg_map)) {
+              printf("expected string but it is not a string\n");
+              break;
+            }
+            size_t len;
+            if (cbor_value_get_string_length(&sys_cfg_map, &len) != CborNoError) {
+              printf("Failed to get the length of the sys_cfg_map key\n");
+              break;
+            }
+            char *key_str = static_cast<char *>(pvPortMalloc(len + 1));
+            configASSERT(key_str);
+            if (cbor_value_copy_text_string(&sys_cfg_map, key_str, &len, NULL) != CborNoError) {
+              break;
```

Comment:
Same comment as above

Reply:
this one is also a bit more for debugging, but this info is a bit more relevant to be printed out. In the end we can deduce from the AGG log what the reportBuilder thought the RBR type was so it may just be better to remove this one as well?

Reply:
removed the potential leak!